spec/lib/query_string_search/matchers/match_attribute_value_spec.rb

```diff
@@ -46,14 +46,14 @@
   describe "build_me?" do
     describe "given a non-nil search_type and search_param" do
       it "is true" do
-        expect(MatchAttributeValue.build_me?(rand, rand)).to be_truthy
+        expect(MatchAttributeValue.build_me?(rand.to_s, rand.to_s)).to be_truthy
```

Comment:
These string conversions are because the method now works only with strings. It's always only ever expected strings, but in the past it didn't require them.